buildtest/cli/__init__.py

```diff
@@ -1006,6 +1006,12 @@ def config_menu(subparsers, parent_parser):
         action="store_true",
         help="Update configuration file with new compilers",
     )
+    compiler_find.add_argument(
+        "-f",
+        "--file",
+        action="store_true",
+        help="Wrtie configuration file to a new file",
+    )
```

Comment:
can you please use [parent parser](https://docs.python.org/3/library/argparse.html#parents) for defining `--file` option so we can potentially use this option in other subcommands.

Reply:
create it as a separate variable see https://github.com/buildtesters/buildtest/blob/869ab652bf5ef2cd5ea77232559574aa3449f82a/buildtest/cli/__init__.py#L226-L229 i think we need to have several parent parsers to help organize where options are inherited. I think we will likely see a clash with option `-f` so rule of thumb when using parent parser only have long option for now, later we can figure out which ones have short option such as `-f`